doc/getting-started.md

```diff
@@ -139,6 +150,46 @@ $ curl -X 'GET' \
 {"_id":"615f30020eb7c3c6616e5ac3","username":"bob","hashed_password":"$2b$12$VtfVij6zz20F/Qr0Ri18O.11.0LJMMXyJxAJAHQbKU0jC96eo2fr.","active":true}
 ```
 
+We can associate available security scopes with an API token. Currently available scopes are 'admin' (admin user permissions) and 'users' (regular user permissions).
```

Comment:
Same comment, the first token should be created with admin scope so that other users and anything else can then be done with the command line tools rather than the raw API.

We may detail parts of the raw API in the full API documentation (in fact that's already generated thanks to the OpenAPI format), but as this is a "getting started" guide it should be as simple as possible with just the minimal steps required to get a system up and running.

Reply:
I have added the detailed description to `api-details` and summary to `getting-started`.